### DISCUSSION TITLE
Caffeine module

### DISCUSSION THREAD

**Discussion by victorballester7** [Jas-SinghFSU/HyprPanel#496] [Ideas]
Hi, 
It's been a few weeks using hyprpanel and so far it's working so well!
But I miss a custom module that I had in my previous waybar config, called `idle_inhibitor`, doing the welll known function of caffeine. Would it be possible to impleemnt it?
My idea would be an icon (like a cafe or an eye for example), that when you click it, it inhibits the idle behaviour and enters in caffeine mode. When you click it again, you disable this feature.
Thanks in advance!

**Comment by rubiin** [comment]
I have a PR , you can check it if you like

**Reply by victorballester7** [reply]
Looks good! Do you know when it will be merged?

**Reply by rubiin** [reply]
not sure.

**Reply by Jas-SinghFSU** [reply]
Been really busy with work but this weekend should afford me some opportunity to review all the pending PRs.

**Reply by Jas-SinghFSU** [reply]
Merged
